backend/app/controllers/api/v1/groups_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class GroupsController < ApplicationController
+      include Dry::Monads::Result::Mixin
+
+      def create
+        case group_creation
+        in Success(group)
+          render json: group, status: :created
+        in Failure(error)
+          render json: error, status: :unprocessable_entity
```

Comment:
You're returning errors here. But information about this you have only here for now.
We need to explicitly tell the frontend what this array we're getting from the backend here is. You could do it via passing a specific key. `errors` in this case.

```ruby
render json: error, status: :unprocessable_entity
```

You could check the [ticket description](https://legolas.atlassian.net/browse/COD-22) more closely and find out by yourself:

> And body of response contains {errors: []}